src/IconButton/_IconButton.scss

```diff
@@ -1,6 +1,6 @@
 $icon-button-colors: (
     "primary": $primary,
-    "secondary": $secondary,
```

Comment:
I worry that people who are used to Bootstrap are gonna try to use "secondary" anyway? Should we alias it to something?